tools/scripts/iceoryx_release.sh

```diff
@@ -41,11 +41,11 @@ done
 target_version=$(head -n 1 "$WORKSPACE/VERSION")
 echo "The new version number will be: $target_version"
 
-find $WORKSPACE -name "*.xml" -type f -exec sed -i 's/<version>\(.*\)<\/version>/<version>'$target_version'<\/version>/g' {} \;
+find "$WORKSPACE" -name "*.xml" -type f -exec sed -i 's/<version>\(.*\)<\/version>/<version>'"$target_version"'<\/version>/g' {} \;
 
-find $WORKSPACE -name "CMakeLists.txt" -type f -exec sed -i 's/set(IOX_VERSION_STRING "\(.*\)")/set\(IOX_VERSION_STRING "'$target_version'"\)/g' {} \;
+find "$WORKSPACE" -name "CMakeLists.txt" -type f -exec sed -i 's/set(IOX_VERSION_STRING "\(.*\)")/set\(IOX_VERSION_STRING "'"$target_version"'"\)/g' {} \;
```

Comment:
Can some of the quotes around `$target_version` be removed?

Reply:
Not sure, the linter proposed me to add some extra double-quotes